features/step_definitions/challenge_promptmeme_steps.rb

```diff
@@ -124,6 +133,13 @@
   step "I fill in Battle 12 challenge options"
 end
 
+When /^I fill in overlapping tags challenge options$/ do 
```

Comment:
It seems like this block could all be pulled up to line 35, similar to how it's 1 step in `challenge_gift_exchange_steps`?